### ISSUE TITLE
normalize_targets=False results in predictions of the same value 

### ISSUE THREAD

**Issue by LilySnow** [DeepRank/deeprank#16]
If I set "normalize_targets=False", the predictions of deeprank for all test models are the same values (except for epoch_0000):

DeepRank output:
[ 0.26067629  0.26067629  0.26067629  0.26067629  0.26067629  0.26067629
  0.26067629  0.26067629  0.26067629  0.26067629  0.26067629  0.26067629
.....]

Here is my code:

> path = '/home/deep/projects/deeprank/BM4/'
> 
> test_files = ['1E6E.hdf5']
> train_database = [ f for f in glob.glob(path + '*.hdf5') if ('1E6E.hdf5' not in f) and ('native' not in f) and ('1
> test_database = [path+f for f in test_files]
> 
> data_set = DataSet(train_database,
>                 test_database = test_database,
>                 grid_shape = (30,30,30),
>                   # select_feature= 'all',
>                 select_feature = {'Feature_ind' : ['pssm_ic']},
>                 pair_chain_feature=np.add,
>                 select_target='DOCKQ',
>                 # does dict_filter filter only  the training data
>                 dict_filter = {'DOCKQ' : '>0.1'},
>                 normalize_features=True,normalize_targets=False,tqdm=False)
> 
> 
> # create the network
> model = NeuralNet(data_set, model3d.cnn,cuda=True,ngpu=1,plot=True)
> 
> # change the optimizer (optional)
> model.optimizer = optim.SGD(model.net.parameters(),
>         lr=0.001,momentum=0.9,weight_decay=0.005)
> 
> # start the training
> model.train(nepoch = 30,train_batch_size = 200,num_workers=8,divide_trainset=[0.8,0.2])
> model.save_model()

**Comment by NicoRenaud** [comment]
When you copy past code you can use the insert code tab so that it s more easily readable. For example

```python
x = np.linspace(0,1,10)
```

**Comment by NicoRenaud** [comment]
That being said where is the script on alembick ?

**Comment by NicoRenaud** [comment]
I can't reproduce that error. That being said not normalizing  the targets during the 'test_learn_reg 3d' in `test_learn.py` leads to divergence of the error ... We might want to enforce target normalization ...
